spec/uktt_api_spec.rb

```diff
@@ -444,4 +444,20 @@
       expect(response[:data].first[:attributes][:quota_order_number_id]).to eq(quota_search_params[:order_number])
     end
   end
+
+  it 'performs a retreive of countries' do
```

Comment:
These are not the tests I want to perform, btw. Ideally I'd do a fully mocked request verification but felt like I needed to at least test something. Coming later will be separately mocked specs for all of the resources catered by the uktt gem